servant-oauth-server.cabal

```diff
@@ -28,14 +28,17 @@ library
   build-depends:       base
                      , aeson >= 2
                      , bytestring
+                     , cryptonite >= 0.24
                      , http-api-data
                      , http-client
                      , http-types
-                     , jose
+                     , jose >= 0.10
```

Comment:
Looks like this version of `jose` is not part of the nix flake.

It was the only external package that `cabal build` built (and not nix). I guess that's due to the version number.

Reply:
Update: The package `jose` has version `0.9` in our nix flake's nixpkgs. I think `jose_0_10`

```nix
  "jose_0_10" = callPackage
    ({ mkDerivation, aeson, base, base64-bytestring, bytestring
     , concise, containers, cryptonite, hedgehog, hspec, lens, memory
     , monad-time, mtl, network-uri, pem, tasty, tasty-hedgehog
     , tasty-hspec, template-haskell, text, time, x509
     }:
     mkDerivation {
       pname = "jose";
       version = "0.10";
       sha256 = "0hhj1a6y7c63vka3f0k732clcx80k74syrfv2r996rxfxmlqlj8z";
       isLibrary = true;
       isExecutable = true;
       libraryHaskellDepends = [
         aeson base base64-bytestring bytestring concise containers
         cryptonite lens memory monad-time mtl network-uri template-haskell
         text time x509
       ];
       testHaskellDepends = [
         aeson base base64-bytestring bytestring concise containers
         cryptonite hedgehog hspec lens mtl network-uri pem tasty
         tasty-hedgehog tasty-hspec text time x509
       ];
       description = "JSON Object Signing and Encryption (JOSE) and JSON Web Token (JWT) library";
       license = lib.licenses.asl20;
       hydraPlatforms = lib.platforms.none;
     }) {};
```

Will add this to my own branch. So, please feel free to ignore this.